### ISSUE TITLE
Automatic CSV Data Import Fails to Populate Odometer Section, Manual Addition Works Properly.

### ISSUE THREAD

**Issue by jackthomasanderson** [hargata/lubelog#317]
Hi,

The bulk import of information via a CSV file fails to automatically update the odometer section, whereas manual input accomplishes this task successfully.

**Comment by hargata** [comment]
Any console errors?

**Comment by hargata** [comment]
Additionally, check the following items:

1. Is your CSV formatted correctly? Note that your CSV file MUST be comma-separated, certain locales generates CSVs that are separated by ";", those need to be converted to "," LubeLogger will only generate and import CSV files separated by ","
2. Do the dates in your CSV file follow your locale? If you're in a locale that uses dd/MM/yyyy your CSV dates MUST be in that format.
3. Do the CSV imports / exports work on other tabs?
4. Always try to reproduce your error on the demo site. Otherwise if I can't reproduce your error I'm not going to be able to fix it.

**Comment by jackthomasanderson** [comment]
Below are my environment variables:

```plaintext
LOGGING__LOGLEVEL__DEFAULT=Error
LC_ALL=fr_FR.UTF-8
LANG=fr_FR.UTF-8
APP_UID=1654
ASPNETCORE_HTTP_PORTS=8080
DOTNET_RUNNING_IN_CONTAINER=true
DOTNET_VERSION=8.0.2
ASPNET_VERSION=8.0.2
```

Regarding my test:

I exported vehicle data to CSV directly from LubeLogger. Then, I cleared all data from my vehicle. Afterward, I imported the previously exported CSV file from LubeLogger. My data was successfully reintegrated, with no error messages in the logs. However, the Odometer section remains empty after the import.

I performed the same test on the demo instance available on the website and encountered the same behavior.

**Comment by hargata** [comment]
so you've exported the Odometer tab data into a CSV file but when you import it it doesn't show up in the Odometer tab? Is that correct? Can you perhaps attach the a portion of the CSV file exported here?

**Comment by hargata** [comment]
Ah! Are you wondering about why the odometer records aren't inserted automatically even if you have Auto-Insert Odometer Record enabled?

**Comment by jackthomasanderson** [comment]
🤣  Yes, that's exactly it!

**Comment by hargata** [comment]
Fixed as of 1.2.2

Watch https://youtu.be/m_FAbZODdyY?si=ErHqjAlsJ5czeDNO&t=29
